safeweb/http_test.go

```diff
@@ -447,7 +447,7 @@ func TestRouting(t *testing.T) {
 			browserPatterns: []string{"/foo/"},
 			apiPatterns:     []string{"/foo/bar/"},
 			requestPath:     "/foo/bar/baz",
-			want:            "multiple handlers match this request",
+			want:            "api",
```

Comment:
@patrickod @awly See if you agree with this change in test expectations. I *think* it's right, given the new behavior, but I have been wrong before. (Once, in 1998. It was traumatizing)

Reply:
LGTM!
In an ideal world, api and non-api muxes shouldn't overlap (except for browser mux serving `/`), but I'm sure there are valid situations that need this.